tests/send-to-helix.proj

```diff
@@ -0,0 +1,101 @@
+<Project Sdk="Microsoft.DotNet.Helix.Sdk" InitialTargets="BuildHelixWorkItems" DefaultTargets="Test">
+
+  <PropertyGroup>
+    <Language>msbuild</Language>
+
+    <_workItemTimeout>00:20:00</_workItemTimeout>
```

Comment:
Just curious, does this include machine availability, or only execution?

Reply:
Only execution on helix. The machine availability would be defined in the yml.